apps/include/apps_ui.h

```diff
@@ -7,8 +7,8 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_APPS_UI_H
-# define HEADER_APPS_UI_H
+#ifndef __OSSL_APPS_APPS_UI_H__
+# define __OSSL_APPS_APPS_UI_H__
```

Comment:
Each name that contains a double underscore __ or begins with an underscore followed by an uppercase letter is reserved to the implementation for any use.

Reply:
So what would you recommend? One underscore on either side or non at all. (Personally, I prefer include guards which are optically clearly distinguishable from ‚ordinary‘ defines. )

Reply:
So what would you recommend? One underscore on either side or non at all. (Personally, I prefer include guards which are optically clearly distinguishable from ‚ordinary‘ defines. )

Reply:
If the define ends with "_H" that is pretty clear already that it is a header guard.
If should start with OSSL_ or OPENSSL_ whatever you prefer.

Reply:
Ok, it was just a ‚test balloon‘. I‘ll remove the surrounding underscores again.

Reply:
> It should start with `OSSL_` or `OPENSSL_` whatever you prefer.

[Richard and I agreed](https://github.com/openssl/openssl/pull/9333#issuecomment-515332651) that it should be `OPENSSL_` for the public headers and `OSSL_` for the internal ones.

Reply:
Underscores [have been removed again](https://github.com/openssl/openssl/compare/e077a760f5e397476c93a4a7857dc429a0f77d78..2de788fd0ea11e340bc1ac18c0c8ef3c081a1f15).